test/mpi/Makefile_common.mtest

```diff
@@ -23,16 +23,8 @@ LDADD += $(top_builddir)/dtpools/src/libdtpools.la
 $(top_builddir)/dtpools/src/libdtpools.la:
 	(cd $(top_builddir)/dtpools && $(MAKE))
 
-if HAVE_GPU
-if GPU_ONLY
-TESTDIRS ?= coll,pt2pt,rma
-TESTLIST ?= testlist.gpu
-else
-TESTLIST ?= testlist,testlist.dtp,testlist.cvar,testlist.gpu
-endif
-else
-TESTLIST ?= testlist,testlist.dtp,testlist.cvar
-endif
+TESTDIRS ?= @TESTDIRS@
+TESTLIST ?= @TESTLIST@
```

Comment:
`Makefile_common.mtest` is not substituted by `configure`. However, I believe `automake` automatically set autocon subst variables. So just deleting these two lines should work.

Reply:
Yeah, I wondered that when I was making this change, but then I looked at lines 15-17 and thought it would be fine 🤷. I'll remove them and try again.

Reply:
I slipped a typo in there at some point. Now I see the problem.